Dockerfile

```
@@ -12,7 +12,7 @@ RUN cargo build --release
 RUN strip target/release/mini-docker-rust
 
 # use a plain alpine image, the alpine version needs to match the builder
-FROM alpine:3.13
+FROM alpine:3.15
```

```suggestion
FROM alpine:3.14
```
The latest rust docker image is still on 3.14.